src/controllers/signup.controller.js

```diff
@@ -0,0 +1,36 @@
+import SignupService from '../services/signup.service';
+import hashPassword from '../helpers/hashPassword';
+
+/**
+ * @exports
+ * @class UsersController
+ */
+class UsersController {
+  /**
+ * users can create an account
+ * @static
+ * @description POST /api/v1/signUp
+ * @param {object} req request object
+ * @param {object} res response object
+ * @memberof UsersController
+ * @returns {object} data
+ */
+  static async signUp(req, res) {
+    const userExist = await SignupService.checkUserExistByEmail(req.body.email);
+    if (!userExist) {
+      const hashedPassword = hashPassword(req.body.password);
+      req.body.password = hashedPassword;
+      const createdUser = await SignupService.addUser(req.body);
+      const {
+        id, firstName, lastName, email, role, isVerified
+      } = createdUser;
+      const data = {
+        id, firstName, lastName, email, role, isVerified
+      };
+      return res.status(201).send({ status: 201, message: 'User created successfully', data });
```

Comment:
Change this to use the response service instead

Reply:
@higustave-ops ☝️

Reply:
He is not seeing the message I have showed him, he is gonna do.